packages/medulas-react-components/src/utils/storybook/index.tsx

```diff
@@ -9,27 +9,27 @@ interface Props {
 
 const globalStyles = makeStyles({
   '@global': {
-    'html, body': {
-      width: '350px',
-      height: '500px',
-      '-ms-overflow-style': '-ms-autohiding-scrollbar',
-      fontSize: '10px',
+    '*': {
+      boxSizing: 'inherit',
+      WebkitFontSmoothing: 'antialiased', // Antialiasing.
+      MozOsxFontSmoothing: 'grayscale', // Antialiasing.
     },
-    'a:-webkit-any-link': {
-      color: 'inherit',
+    '*::before, *::after': {
+      boxSizing: 'inherit',
+    },
+    html: {
+      fontSize: '62.5%',
     },
     body: {
-      bottom: 0,
-      top: 0,
-      left: 0,
-      right: 0,
+      margin: '0',
+      padding: '0',
+      bottom: '0',
+      top: '0',
+      left: '0',
+      right: '0',
       overflowX: 'hidden',
-      fontFamily: "'Muli', sans-serif",
-      margin: 0,
-      backgroundColor: theme.palette.background.default,
```

Comment:
Why removing this as well?

Reply:
I just get it from `bierzo-wallet` storybook config